### PR TITLE
Update theme

### LINKED ISSUE #4: Fix layout on mobile
It's not great right now, especially on my iPhone mini! We probably need to make the two-column design work more responsively, maybe putting the tags list behind a hamburger menu?

### LINKED ISSUE #81: button text missing on ios?
On the Save Bookmark screen, the submit button doesn't have any visible text.

### PR DESCRIPTION
**Pull request by johnholdun**
(This builds on #120 and #122 so please review just the last commit. I'll rebase as those are updated and merged. Thanks!)

Adds some enhancements to the theme/UX. There are some DX changes as well, like making sure all the colors in the CSS file use variables. This is mostly subjective, so if there's anything I should alter or roll back, please don't hesitate to ask!

See it live here, and ask me privately for the password if you want to see the admin stuff: https://befitting-innate-roadway.glitch.me

[ed. note: fixes #81 and fixes #4 btw! -ckolderup]

### CODE REVIEW

File: src/pages/layouts/main.hbs
Comment (johnholdun) [inline src/pages/layouts/main.hbs:44]: I merged the main and admin templates into one—they were mostly the same save for the admin subnav showing up conditionally. I mad that condition a parameter of the main theme.

**Comment by andypiper** [comment]
Just an observation on the test site - this looks *really* nice, and as a bonus, seems to also resolve the weirdness that causes the buttons to show as white text on white on my iOS browser for the existing default theme. Nice work!

**Comment by ckolderup** [comment]
We talked about this a bit on IRC today but yeah this is looking _crisp_ @johnholdun, thank you, and it's a big relief to see things resizing properly on mobile as well. 

If you have a chance to get that admin nav stuff appearing before the admin page content like we discussed I think I'd be very happy to merge this along with the first PR.

**Comment by ckolderup** [comment]
if I say fixes #81 in a comment, will it link things properly? I use Jira at work, sob

**Comment by andypiper** [comment]
Oh, is there an IRC?

**Comment by ckolderup** [comment]
not right now! John and I have just been in a group chat that never left IRC, so it's our primary way of talking to each other.

**Comment by johnholdun** [comment]
Added another commit here that changes the order of the admin nav to appear before main content on small screens, _and_  makes the admin links programmatic so that the link to the current page is unlinked and gets a different visual treatment (on all screen sizes). The changes are on the Glitch if you want to see.

I squashed #123 into the first commit here and forced-push so I think that this can be merged after that one without conflicts, but I'll re-rebase if that ends up not being true.

**Comment by ckolderup** [comment]
it appears to be not true, sorry. I'll get committer access set up for you & the other current contributors soon and then we can use branches in the repo that we can all collaborate on more easily.

**Comment by johnholdun** [comment]
Haha I messed that up so bad. Rebased again though!

**Review by ckolderup — APPROVED** [review]

**Comment by ckolderup** [comment]
thanks! I'm so happy to have a mobile-friendly view now 😊

**Pull request merged**
